api/problem_details.py

```diff
@@ -237,4 +237,11 @@
     status_code=409,
     title="Incompatible genre.",
     detail="The genre is incompatible with the fiction status of the work."
+)
+
+EROTICA_FOR_ADULTS_ONLY = pd(
+    "http://librarysimplified.org/terms/problem/erotica-for-adults-only",
+    status_code=409,
```

Comment:
400 might be better for these since the things that conflict are all in the same request